.github/workflows/deploy-production.yml

```
@@ -111,19 +108,19 @@ jobs:
 
       - name: Install infrastructure dependencies
         working-directory: infrastructure
-        run: pnpm install --frozen-lockfile
+        run: npm ci
 
       - name: Verify Pulumi stack config
         working-directory: infrastructure
         run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:2:25: Double quote to prevent globbing and word splitting

(shellcheck)